jbpm-services/jbpm-executor/src/main/java/org/jbpm/executor/impl/ExecutorImpl.java

```diff
@@ -56,12 +65,20 @@
     
     private ExecutorStoreService executorStoreService;
 
-	private List<ScheduledFuture<?>> handle = new ArrayList<ScheduledFuture<?>>();
+    private List<ScheduledFuture<?>> handle = new ArrayList<ScheduledFuture<?>>();
     private int threadPoolSize = Integer.parseInt(System.getProperty("org.kie.executor.pool.size", "1"));
     private int retries = Integer.parseInt(System.getProperty("org.kie.executor.retry.count", "3"));
     private int interval = Integer.parseInt(System.getProperty("org.kie.executor.interval", "3"));
     private int initialDelay = Integer.parseInt(System.getProperty("org.kie.executor.initial.delay", "100"));
     private TimeUnit timeunit = TimeUnit.valueOf(System.getProperty("org.kie.executor.timeunit", "SECONDS"));
+    
+    
+    // jms related instances
+    private boolean useJMS = Boolean.parseBoolean(System.getProperty("org.kie.executor.jms", "true"));
+    private String connectionFactoryName = System.getProperty("org.kie.executor.jms.cf", "java:/JmsXA");
+    private String queueName = System.getProperty("org.kie.executor.jms.queue", "queue/KIE.EXECUTOR");
```

Comment:
yup, will describe this in docs when merged but will add just few lines of comments here any way